sdk/app_cpu1/common/drv/timing_manager.c

```diff
@@ -67,25 +69,27 @@ int interrupt_system_init(void)
 
 /*
  * Initialize the timing manager driver. This initializes
- * the interrupt system and sets the default PWM event qualifier
+ * the interrupt system and sets the default configurations
  */
 void timing_manager_init(void)
 {
     printf("TIMING MANAGER:\tInitializing...\n");
-    // Initialize interrupts
+    // Initializes the interrupt used to
     int result = 0;
-    result = interrupt_system_init();
-
+    result = timing_manager_interrupt_system_init();
     if (result != XST_SUCCESS) {
         printf("Error initializing interrupt system.");
     }
 
+    // Set the interrupt source for the scheduler based on user configuration settings
+    timing_manager_set_scheduler_source();
+
     // Set the timing manager to automatic triggering
     // (this call is redundant, as the FPGA should reset slv_reg0 to 0x0000_0001)
     timing_manager_set_mode(TM_AUTOMATIC);
 
     // Default event qualifier is PWM carrier high AND low
-    timing_manager_trigger_on_pwm_both();
+    timing_manager_trigger_on_pwm_low();
```

Comment:
Hmm... this doesn't match the comment. Did the default change from `BOTH` to `LOW`, and the comment should be updated to match? Or was this a thing you switched for debugging and didn't change back?

Reply:
You're right, I totally missed that. I made the default low as per #377, "The ISR should only fire on the bottom of the PWM triangle carrier." - so comment needs to be updated